tests/console/a2ps.pm

```diff
@@ -0,0 +1,14 @@
+use strict;
+use base "consoletest";
+use testapi;
+use ttylogin;
+
+sub run() {
+    become_root;
+    assert_script_run("zypper -n in a2ps | tee /dev/$serialdev");
```

Comment:
if you use a pipe, the assert_script_run is useless - the exit status will be very likely correct as the tee is pretty easy to please